bundles/EcommerceFrameworkBundle/IndexService/Worker/ProductCentricBatchProcessingWorker.php

```diff
@@ -203,7 +203,7 @@ protected function getDefaultDataForIndex(IndexableInterface $object, $subObject
 
         $virtualProductId = $subObjectId;
         $virtualProductActive = $object->isActive();
-        if ($object->getOSIndexType() == 'variant') {
+        if ($object->getOSIndexType() == DataObject::OBJECT_TYPE_VARIANT) {
```

Comment:
`ProductListInterface::PRODUCT_TYPE_VARIANT` ?

Reply:
I was unsure, but you are probably correct. Fixing.